### ISSUE TITLE
Modify my portfolio website.

### ISSUE THREAD

**Issue by namruthahari** [namruthahari/firebase-host-web#1]
Add animations, social media icons, hover effects. Try to make this website cool.

**Comment by mifrasgaffoor** [comment]
Hey @namruthahari  i am searching for projects and issues for my very first hacktoberfest contributions .... can you please assign it to me .
Thanks in advance!

**Comment by namruthahari** [comment]
Sure @mifrasgaffoor. Let me know what you will be working on this project! Welcome to hacktoberfest!

**Comment by mifrasgaffoor** [comment]
Thank you @namruthahari could I design a contact form?
I am a  beginner learning new things nowadays.

**Comment by namruthahari** [comment]
sure! Make a new page for contact form!

**Comment by Agasthyan07** [comment]
hai
can I make you a good top navbar for your webpage?

**Comment by namruthahari** [comment]
Sure @Agasthyan07. D attach screenshots along with your pull request. Your code will be reviewed.

**Comment by GayanKod** [comment]
Hello there! Can I know Is this project hacktoberfest accepted?

**Comment by GayanKod** [comment]
I can make this website more professional

**Comment by IAmSalvaMartini** [comment]
> Hello there! Can I know Is this project hacktoberfest accepted?

Yes. Go ahead and submit your work, we'll review it and accept it

**Comment by GayanKod** [comment]
Okay I will thank you so much

**Comment by oneamanish** [comment]
see my pull req.
